### DISCUSSION TITLE
Ability to view repo collaborators (and other data) without having admin access to repo (UI and API)

### DISCUSSION THREAD

**Discussion by dja852** [community/community#62413] [Enterprise]
### Select Topic Area

Product Feedback

### Body

Hi! 

In many companies, there is a business need for some users (think Compliance or Internal Audit teams) to be able to view collaborators and associated permissions on repos (+ other repo data like protected branch setup), without having write or admin access to those repos themselves.

I wanted to propose an additional enterprise/organisation-level role that when granted to these users will allow the collaborators and their associated permissions to be viewed for each repo on both the web and API (+ other repo data). This would also be useful to assign to service-related accounts to allow for read-only integrations without requiring those accounts needing write/admin permissions.

This is in the context of GitHub Enterprise Cloud. Thanks!

**Comment by mharrah** [comment]
Agreed - we could really benefit from having a role we could assign to compliance/security/internal audit to view all the settings and permissions and roles that an admin can see, but can't change them.

As it stands now, we regularly have to record screencap videos pulling up data on the creen to prove who can commit to a repo, or who can merge a pull request, or who can create a release.

**Comment by sloperun3az** [comment]
This is a fun game, and if you play it with some pals, you're going to have a great time. One of my very good pals who is currently employed at **<a href="https://cookie-clicker.io/">cookie clicker</a>** is the one who introduced me to this game, and I've had the opportunity to collaborate with him on quite a few occasions when playing it. It is really incredible!
